fjord/feedback/models.py

```diff
@@ -750,18 +751,14 @@ def purge_data(cutoff=None, verbose=False):
     responses_to_update.update(objs.values_list('opinion_id', flat=True))
     count = objs.count()
     objs.delete()
-
-    if verbose:
-        print 'Purged %d feedback_responseemail records' % count
+    msg = 'feedback_responseemail: %d, ' % (count, )
 
     # Second, ResponseContext.
     objs = ResponseContext.objects.filter(opinion__created__lte=cutoff)
     responses_to_update.update(objs.values_list('opinion_id', flat=True))
     count = objs.count()
     objs.delete()
-
-    if verbose:
-        print 'Purged %d feedback_responsecontext records' % count
+    msg += 'feedback_responsecontext: %d, ' % (count, )
```

Comment:
Not the best way to construct a string but does with minimal changes to the existing code